app/controllers/v0/sign_in_controller.rb

```diff
@@ -342,6 +337,10 @@ def idme_auth_service(type)
         @idme_auth_service
       end
     end
+   
+    def mocked_authentication_auth_service
```

Comment:
Could we change this to match the other auth services? 
```suggestion
    def vamock_auth_service
```